apis/fluent-en_GB/atrium-api-fluent-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/pathAssertions.kt

```diff
@@ -312,30 +312,26 @@ fun <T : Path> Expect<T>.isRelative(): Expect<T> =
     _logicAppend { isRelative() }
 
 /**
- * Expects that the subject of the assertion (a [Path]) is a directory;
- * meaning that there is a file system entry at the location the [Path] points to and that is a directory.
+ * Expects that the subject of the assertion (a [Path]) is a directory having the provided entries.
+ * That means that there is a file system entry at the location the [Path] points to and that it is a directory.
+ * Furthermore, every argument string resolved against the subject yields an existing file system entry.
  *
- * Every argument string is expected to exist as a child file or child directory for the subject of the assertion.
- *
- * This assertion _resolves_ symbolic links.
- * Therefore, if a symbolic link exists at the location the subject points to, search will continue
- * at the location the link points at.
+ * This assertion _resolves_ symbolic links for the subject, but not for the entries.
+ * Therefore, if a symbolic link exists at the location the subject points to, the search will continue at the location
+ * the link points at. If a symbolic link exists at one of the entries, this will fulfill the respective assertion and
+ * the entry’s symbolic link will not be followed.
  *
- * This assertion is not atomic with respect to concurrent file system operations on the paths the assertions works on.
- * Its result, in particular its extended explanations, may be wrong if such concurrent file system operations
+ * This assertion is not atomic with respect to concurrent file system operations on the paths the assertions work on.
```

Comment:
Could you please fix this typo also in other files?

Reply:
Done.